contrib-04-HOP-LoKi-Functor/ProceedingHOP.md

```diff
@@ -116,7 +92,7 @@ To check their performance both tools are applied to \BdKstee and \BKemu simulat
 
 # Conclusions
 
-The concept of the HOP mass introduced in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en) is implemented in the \lhcb software by means of a LoKi functor and a TupleTool. The technical details of the implementation and usage of the tools are described both to document them and also to serve as a guide on how to proceed when developing this kind of tools for the first time. Then the results are compared to those obtained for the studies in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en). A perfect agreement is found proving that the tools are working as expected and ready to be incorporated to the \lhcb software.
+The concept of the HOP mass introduced in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en) is implemented in the \lhcb software by means of a LoKi functor and a TupleTool. The technical details of the implementation and usage of the tools are described both to document them and also to serve as a guide on how to proceed when developing this kind of tools for the first time. Then the results are compared to those obtained for the studies in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en). A perfect agreement is found, so the tools have been added to the \lhcb software. The LoKi functor and the TupleTool have been incorporated into the `Phys/LoKiPhys` and the `Phys/DecayTreeTuple` packages respectively.
```

Comment:
Disagree for two reasons:

1. The repo is not really for this (it's the proceedings) and it becomes outdated as fast as the note.
2. I'm not saying we add the whole code, just how to use the tools.

I think I was not clear on what I was asking. It would be something like

```python
dtt = DecayTreeTuple('Bu2LLK_ee')
...
dtt.Decay = '[B0 -> K*(892)0 (J/psi(1S) -> e- e+ )]CC'
dtt.addBranches({'B0'    : '[^(B0 -> K*(892)0 (J/psi(1S) -> e- e+ ))]CC'})
b0_hybrid = dtt.B0.addTupleTool('LoKi::Hybrid::TupleTool/LoKi_B0')
b0_hybrid.Variables = {
    'hop_mass': 'BPVHOPM',
    'corr_mass': 'BPVCORRM'
}
```

with an annotation of the versions of software where it was working and the fact that some specific configurations are missing. In this way, if people what to use the tools they know how to (it's trivial, but you are documenting their use).